terminology.md

```diff
@@ -232,131 +264,154 @@ function memoizedFib(num, memo = {}) {
 [ECMA262 (symbols)](https://tc39.es/ecma262/#sec-well-known-symbols)
 
 ## Primordial
 
 ### Definition
+
 An [intrinsic](#Intrinsic) value that is accessible to ECMAScript code and required to exist before any ECMAScript code runs.
 
 ### Example
+
 %Array%, the initial value of a realm's Array constructor, is accessible as `Array`.
 
 ## Effectively undeniable
 
 ### Definition
+
 A [primordial](#Primordial) value that is accessible to ECMAScript code without reference to named bindings other than those for prototype and property descriptor reflection (i.e., solely by syntax, `__proto__`, and primordial `getPrototypeOf`/`getOwnPropertyDescriptor`/`getOwnPropertyDescriptors` functions).
 
 ### Example
+
 %Array.prototype%, the initial value of a realm's Array prototype, is accessible as `[].__proto__`.
 %ThrowTypeError%, a realm's special TypeError-throwing function, is accessible as `(function(){ 'use strict'; return Object.getOwnPropertyDescriptor(arguments, 'callee').get; })()`.
 
 ## Normative
 
 ### Definition
+
 Statements that constrain the observable behavior of implementations (i.e., the behavior that conforming implementations are allowed to exhibit).
 
 ## SES
 
 ### Definition
+
 Secure ECMAScript. A subset of ECMAScript.
 
 ### References
+
 [Useful diagram](https://github.com/Agoric/Jessie/blob/master/README.md)
 
 ## Plenary
 
 ### Definition
+
 Meeting of all available TC39 delegates. Occurs around 6 times a year.
 
 ## Refactoring hazard
 
 ### Definition
+
 A seemingly simple change (i.e. thought to be semantically equivalent) that seems like a drop in replacement for an existing piece of code but has specific edge cases that make it not work as expected. As you refactor code from "old pattern" to "new pattern", it would be easy to cause unintentional effects (bugs, etc).
 
 ## Reify
 
 ### Definition
+
 To make something a first-class concept in a language specification, such that it can be passed around, inspected, or manipulated by user code.
 
 ### Example
+
 - The C programming language reifies the concept of memory addresses with pointers.
 - ECMAScript reifies its own interpreter with `eval`.
 
 ### References
+
 - [Reification](https://en.wikipedia.org/wiki/Reification_(computer_science))
 
 ## SDO (2 definitions)
 
 ### Definition (1)
+
 SDOs are syntax-directed operations in the ECMA-262 specification. These are the operations that are defined piecewise over the parse tree rather than by having a single algorithm. In other terms it is a particular style of algorithm in the specification.
 
 ### Example (1)
+
 Evaluation is an SDO because Evaluation is defined piecewise over several productions.
 
 ### References (1)
-https://tc39.es/ecma262/multipage/notational-conventions.html#sec-algorithm-conventions-syntax-directed-operations
+
+<https://tc39.es/ecma262/multipage/notational-conventions.html#sec-algorithm-conventions-syntax-directed-operations>
```

Comment:
These are the only two raw links, so it seems better to just give them link text:
```suggestion
[Specification](https://tc39.es/ecma262/multipage/notational-conventions.html#sec-algorithm-conventions-syntax-directed-operations)
```